src/atoms/Modal/Modal.js

```diff
@@ -16,6 +16,8 @@ type ModalState = {
   blurred: boolean,
 };
 
+const MODAL_BODY_CLASS = 'modal-shown';
```

Comment:
Where are the styles of this class?

Reply:
@ej9x are you think we should hardcode styles with `.modal-shown > #page` selector?

Reply:
@zouxuoz I mean about adding blur css styles to the boost.